docs/doxygen/logging/logging.dox

```diff
@@ -0,0 +1,45 @@
+/**
+@page logging Logging Infrastructure
+@brief This page contains an overview of the logging infrastructure used by the C-SDK Libraries.
+
+<p>
+The C-SDK libraries utilize one of the following 4 macro interfaces for logging within the library for different verbosity levels:
+- `LogError` for **Error** level
+- `LogWarn` for **Info** level
+- `LogInfo` for **Info** level
+- `LogDebug` for **Debug** level
+
+**Note: By default, each library contains empty definitions of the above logging macros, thereby, generating no code for logging calls.**
+
+To enable logging within the libraries, the customer application should provide implementation of these macros through the library-specific config file (for example, the `core_mqtt_config.h` file for MQTT library).
+Implementation of these macros can be provided as mapping to the customer's platform-specific logging implementation.
```

Comment:
I'd try to avoid using "implementation" twice in this sentence (e.g. "logging framework"?)
```suggestion
The implementation of these macros can be provided as a mapping to the customer's platform-specific logging framework.
```